deeptime/markov/msm/tram/_tram_dataset.py

```diff
@@ -21,6 +19,30 @@ def _determine_n_therm_states(dtrajs, ttrajs):
         return _determine_n_states(ttrajs)
 
 
+def transition_counts_from_count_models(n_therm_states, n_markov_states, count_models):
+    transition_counts = np.zeros((n_therm_states, n_markov_states, n_markov_states), dtype=np.int32)
+
+    for k in range(n_therm_states):
+        model_k = count_models[k]
+        if model_k.count_matrix.sum() > 0:
+            i_s, j_s = np.meshgrid(model_k.state_symbols, model_k.state_symbols)
+            # place submodel counts in our full-sized count matrices
+            transition_counts[k, i_s, j_s] = model_k.count_matrix.T
```

Comment:
why do you have to transpose here? because of how the meshgrid is organized? seems like a good source for errors :-D

Reply:
Actually this was an error already :D there is a unit test for it. I changed indexing of the meshgrid so the transpose is gone.